models/public/index.md

```diff
@@ -154,6 +154,12 @@ SSD-based and provide reasonable accuracy/performance trade-offs.
 | YOLO v4                              | Keras\*                  | [yolo-v4-tf](./yolo-v4-tf/README.md) | 71.17%/75.02% | 128.608	 |	64.33 |
 | YOLO v4 Tiny                         | Keras\*                  | [yolo-v4-tiny-tf](./yolo-v4-tiny-tf/README.md) | | 6.9289 | 6.0535 |
 
+## Face Detection
```

Comment:
We don't have a separate face detection task type; this section should be merged into Object Detection.

Note that you can find a model's task type in its `model.yml` or `composite-model.yml` file.